src/concrete/ml/quantization/quantized_module.py

```diff
@@ -90,6 +92,8 @@ class QuantizedModule:
     input_quantizers: List[UniformQuantizer]
     output_quantizers: List[UniformQuantizer]
     fhe_circuit: Union[None, Circuit]
+    _onnx_preprocessing: Optional[onnx.ModelProto] = None
```

Comment:
these attributes should not be class attributes but class instances, so better to initialize them in the __init__ 

also, I would add a property setter for both instead of having to access these protected attributes in tests and so on

Reply:
Yes but that's true for all of them I suppose. We can do some refacto to clean this.

Agreed on the setter!

Reply:
see above (https://github.com/zama-ai/concrete-ml/pull/778/files/51d0d124b199c50bb81154dfd11811e352527ede#r1670354637): I think it makes sense to include `_onnx_preprocessing` in the init 

see below (https://github.com/zama-ai/concrete-ml/pull/778/files/51d0d124b199c50bb81154dfd11811e352527ede#r1670359773) : I think one of these 2 attributes could be removed for simplicity

Reply:
Ok for the first point but I not sure about the second. If you think we shouldn't return onnx + numpy forward for the preprocessing then we should do the same for the equivalent numpy forward, no?